unpublish.php

```diff
@@ -123,7 +124,7 @@ public function render_unpublish_ui() {
 		$unpublish_timestamp = $this->get_unpublish_timestamp( get_the_ID() );
 		if ( ! empty( $unpublish_timestamp ) ) {
 			$local_timestamp = strtotime( get_date_from_gmt( date( 'Y-m-d H:i:s', $unpublish_timestamp ) ) );
-			$datetime_format = sprintf( __( '%s @ %s', 'unpublish' ), $this->date_format, $this->time_format );
+			$datetime_format = __( 'M j, Y @ H:i', 'unpublish' );
```

Comment:
Translators note would be handy here, again borrowing from the core code. 
`/* translators: Unpublish box date format, see https://secure.php.net/date */`